gphotos/GoogleAlbumsSync.py

```diff
@@ -79,7 +79,12 @@ def fetch_album_contents(self, album_id: str,
             media_json = items_json.get('mediaItems')
             # cope with empty albums
             if not media_json:
-                break
+                if not items_json.get('nextPageToken'):
+                    break
+                else:
+                    media_json = []
```

Comment:
Setting media_json to an empty array allows for less changes to the codebase.  The forloop will see an empty array instead of trying to iterate over a NoneType.